src/main/java/org/selenium/pages/ProductDetailPage.java

```diff
@@ -0,0 +1,52 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.By;
+
+import java.util.Locale;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class ProductDetailPage extends BasePage {
+    private By waitContainer = By.xpath("//div[@class='primary_block row']");
+    private By waitPopup = By.xpath("//div[@class='fancybox-overlay fancybox-overlay-fixed']");
+    private By buttonAddWish = By.xpath("//a[@id='wishlist_button']");
+    private By productName = By.xpath("//h1[@itemprop='name']");
+    private By textPopup = By.xpath("//p[@class='fancybox-error']");
+    private By closeButton = By.xpath("//a[@title='Close']");
+    private By account = By.xpath("//a[@title='View my customer account']");
+
+    public String getNameOfProductExpected() {
```

Comment:
Same comment regarding name as for MyAccountPage

Reply:
Fixed it